test/data/mock.php

```diff
@@ -213,7 +213,6 @@ protected function cspLog( $req ) {
 
 	protected function cspClean( $req ) {
 		file_put_contents( $this->cspFile, '' );
-		unlink( $this->cspFile );
```

Comment:
What's the reason for this change? It's meant to clean up after the test so it should stay.